src/packages/solid/components/Key/Key.tsx

```diff
@@ -46,9 +51,12 @@ const Key: Component<KeyProps> = props => {
       style={styles.Container}
       width={
         styles.Container.sizes[props.size || 'sm'] * styles.Container.baseWidth +
```

Comment:
My head just says all this sizing is overkill - options people will never use... (prove me wrong)